### PR TITLE
Theme customization

### PR DESCRIPTION
**Pull request by ctf-umbraco**

### CODE REVIEW

File: Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+versionFrom: 8.0.0
+---
+
+# Customize Your theme
+
+Your theme lets you add your touch to your page, meaning the colors you want, the font, and all that. firstly if you are new to working with Umbraco Uno and do not yet know how to work with the [Theme](../../Getting-Started/Themes/index.md) then it is recommended that you take a look.
```
Comment (Jay-umbr) [inline Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md:7]:
```suggestion
A custom theme lets you add a little personal touch to your page - the colors you want, the font, and all that. If you are new to working with Umbraco Uno and do not yet know how to work with the theme then it is recommended that you [take a look.](../../Getting-Started/Themes/index.md)
```

File: Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## How to customize your theme through the theme editor
+
+We will be working with the theme editor, you can access this by going to the preview section of the unicorn hotel page.
```
Comment (Jay-umbr) [inline Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md:11]:
```suggestion
We will be working with the theme editor, , which you can access by going to the preview section of the _Unicorn Hotel_ page.
```

File: Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+### Picking the colors and text of the theme
+
+First what we will need to do is click the green button that looks like a paintbrush, then the theme editor will show up.
```
Comment (Jay-umbr) [inline Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md:15]:
```suggestion
First what we will need to do, is click the green button that looks like a paintbrush - then the theme editor will show up.
```

File: Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+9. Nav Padding Small Screen 15
+
+Then navigate to the footer tab:
+
+1. Choose Dark Theme
+2. Select Container Big
+
+## Logo
+
+To set up the logo we will have to go back to the backoffice and got into settings and find Theme.
```
Comment (Jay-umbr) [inline Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md:54]:
```suggestion
To set up the logo we will have to go back to the backoffice, go into settings and find Theme.
```

File: Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+---
+versionFrom: 8.0.0
+---
+
+# Customize Your theme
+
+A custom theme lets you add a little personal touch to your page - the colors you want, the font, and all that. If you are new to working with Umbraco Uno and do not yet know how to work with the theme then it is recommended that you [take a look.](../../Getting-Started/Themes/index.md)
```
Comment (sofietoft) [inline Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md:7]:
```suggestion
A custom theme lets you add a little personal touch to your page - the colors you want, the font, and all that. If you are new to working with Umbraco Uno and do not yet know how to work with the theme then it is recommended that you [checkout the Theme Editor.](../../Getting-Started/Themes/index.md)
```

File: Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+## How to customize your theme through the theme editor
+
+We will be working with the theme editor, , which you can access by going to the preview section of the _Unicorn Hotel_ page.
```
Comment (sofietoft) [inline Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md:11]:
```suggestion
We will be working with the theme editor, which you can access by **previewing** any page on your website.
```

File: Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
+---
+versionFrom: 8.0.0
+---
+
+# Customize Your theme
```
Comment (sofietoft) [inline Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md:5]:
```suggestion
# Customize your theme
```

File: Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+### Picking the colors and text of the theme
+
+First what we will need to do, is click the green button that looks like a paintbrush - then the theme editor will show up.
+The theme we will be setting up will be a watermelon theme, and it will look like this:
+
+![watermelon theme](images/theme.gif)
+
+:::note
+If you want a colored theme you will have to go to the Hero Widget and set the overlay to 0 otherwise the color you choose for background color will be put on top of your hero image
```
Comment (sofietoft) [inline Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md:21]:
```suggestion
If you want to use colorful background colors it is recommended that you do not configure the *Overlay Strength* feature on your widgets, as the colors will be used for the overlay.
```

File: Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+::::
+
+1. Select ***Theme*** and press in the field and type in the value you see on this image:
```
Comment (sofietoft) [inline Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md:24]:
```suggestion
1. Type in the values you see on this image below or choose your own color scheme.
```

File: Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+    ![watermelon theme color](images/Color-settings.png)
+
+2. Do this for all the fields and press ***Save***
+3. This will result in the color Scheme for the dark theme to be set up like intended.
+4. Select ***Text*** in the menu bar at the top of the theme editor.
+5. Choose Roboto as both Heading and Body Font
```
Comment (sofietoft) [inline Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md:31]: Removed item number 2, as I've suggested to write the first bullet.
Removed item number 3, as it wasn't really a step, but more a statement.

```suggestion
2. Select ***Text*** in the menu bar at the top of the theme editor.
3. Choose Roboto as both Heading and Body Font
```

File: Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+
+### Setting up Header and Footer
+
+Firstly navigate to the header tab in the theme editor, then follow the process below:
```
Comment (sofietoft) [inline Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md:35]:
```suggestion
Firstly navigate to the Header tab in the theme editor, then follow the process below:
```

File: Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+2. Set Nav Position Left
+3. Set Container to Full Width
+4. Height should be 60
+5. Height Mobile should be 50
+6. Logo Padding should be left at 20
+7. Logo Padding Mobile should be set to 16
+8. Nav Padding set to 30
+9. Nav Padding Small Screen 15
+
+Then navigate to the footer tab:
```
Comment (sofietoft) [inline Umbraco-Uno/Creating-Content/Customize-Your-Theme/index.md:47]:
```suggestion
Then navigate to the Footer tab:
```

**Pull request merged**